lms/djangoapps/courseware/tests/test_course_info.py

```diff
@@ -114,3 +115,34 @@ def test_anonymous_user_xml(self):
         resp = self.client.get(url)
         self.assertEqual(resp.status_code, 200)
         self.assertNotIn(self.xml_data, resp.content)
+
+
+@attr('shard_1')
+@override_settings(FEATURES=dict(settings.FEATURES, EMBARGO=False))
```

Comment:
Note: the embargo middleware was making this test unreliable in Jenkins due to caching of query results. Turning embargo off allows this test to pass reliably both locally and in CI.